app/controllers/api/api_docs_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Api::ApiDocsController < Api::BaseController
+  before_action :find_service
+  before_action :find_service_apis_collection, only: :new
+  before_action :activate_submenu
+
+  prepend_view_path 'app/views/admin/api_docs/services/'
```

Comment:
@hallelujah I didn't know about this and I've done for:
![image](https://user-images.githubusercontent.com/11318903/46637771-fc4b7200-cb5d-11e8-8e5b-caac2b24ade5.png)
But I am not sure it is better 🤔 maybe so, but it requires specifying the `render '/index'` and `render '/index'` anyway because otherwise it checks under `'app/views/admin/api_docs/services/'api/api_docs/...`. And it also required changing a view (look the commit for this in https://github.com/3scale/porta/pull/72/commits/245093a19ef8d7e470c1f23ec2ca176da12a5f5f)

Reply:
Ah right! Sorry! 🤦‍♂️

Reply:
👍 😉 
Done (it is just removing that specific commit 😄 ) but I haven't pushed yet. Any other change that I should know before pushing? 😝